service/src/test/java/bio/terra/pipelines/stairway/imputation/StairwayExceptionSerializerTest.java

```diff
@@ -83,15 +84,16 @@ void deserialize_classNotFound() {
 
   @Test
   void deserialize_failToConstruct() {
-    // this is marked as an ApiErrorReportException, but no matching constructor can be found
+    // this is marked as an ApiErrorReportException, but no matching constructor can be found.  We
+    // dont throw this in our stairway flgiht code so it doesnt need to be deserializable
```

Comment:
I think I understand this after thinking about it for a while - to help future devs understand this more easily, can you add some comment here to clarify that we do throw a `LeonardoApiServiceException` but not a `LeonardoServiceException` and why those are different?

Reply:
updated the comment!